File: Day8_RemoveOddPlaceCharacters/main.cpp

#include <iostream>

using namespace std;

int
main ()
{
  string original = "Technology Encroyable";
  string result = "";

  for (int i = 0; i < original.length (); i++)
    {
      if (i % 2 == 0)
	{
	  result += original[i];
	}
    }
  cout << "Original String" + original + "\n";
  cout << "New String" + result;

  return 0;
}

File: Day8_RemoveOddPlaceCharacters/main.py
#remove characters from
# odd indexes [PYTHON]

str = "Technology Encroyable"
newStr = ""
i = 0 
while i<len(str):
    if(i%2==0):
        newStr += str[i]
        i += 1 
    i+= 1 
          
print(newStr)    
       
